Util/GameParser.gd

```diff
@@ -76,6 +76,11 @@ func callObjectFuncWrapper(_obj: String, _command: String, _args: Array):
 		
 	if(object == null):
 		return "[color=red]!RUNTIME ERROR NO CHARACTER FOUND "+_obj+"."+_command+" "+str(_args)+"![/color]"
+	if(_command == "theyre" && _args.size() == 0):
+		if(object.isPlayer()):
+			return "you're"
```

Comment:
I don't like the default cause for the player there, especially for the:

```diff
-			"Look at you, parading that bitch around like she’s yours.",
+			"Look at you, parading that bitch around like {target.theyre} yours.",
```
and
```diff
-			"I see we have a nosy little [[SLUT]] here, trying to see what she’s not supposed to.",
+			"I see we have a nosy little [[SLUT]] here, trying to see what {imate.theyre} not supposed to.",
```
Where not using "your're" is not intended, I think just removing the player check would be fine.